src/tests/ftest/scripts/main.sh

```diff
@@ -190,6 +190,20 @@ fi
 
 export DAOS_APP_DIR=${DAOS_APP_DIR:-$DAOS_TEST_SHARED_DIR}
 
+# Create two test users in the same group
+if ! ./user_setup.py -n "${TEST_NODES}" -g daos_test_groupx -u daos_test_userx1,daos_test_userx2 -s; then
+    exit "${PIPESTATUS[0]}"
+else
+    rc=0
+fi
+
+# Create a third test user in a different group
+if ! ./user_setup.py -n "${TEST_NODES}" -g daos_test_groupy -u daos_test_usery1 -s; then
```

Comment:
Optional readability improvement:
```suggestion
if ! ./user_setup.py -n "${TEST_NODES}" -g daos_test_group_y -u daos_test_user_y1 -s; then
```